pkg/skaffold/plugin/builders/jib/local_gradle_test.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jib
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+	"github.com/docker/docker/api/types"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/pkg/errors"
+)
+
+type mockLocalDaemon struct {
```

Comment:
Instead of creating a new `mockLocalDaemon`, I think you can use our [FakeAPIClient](https://github.com/GoogleContainerTools/skaffold/blob/master/testutil/fake_image_api.go#L32) to mock out the docker daemon, which should make this test a bit simpler. 

An example unit test that uses it and calls `ImageID` (which seems to be the function you need for this test) can be found [here](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/docker/image_test.go#L126).

Reply:
Will do so, thanks!